src/redux/categories/Categories.js

```diff
@@ -0,0 +1,13 @@
+export const checkStatus = () => (
+  {
+    type: 'CHECK_STATUS',
+    payload: 'Test',
+  }
+);
+
+export default function categoriesReducer(state = [], action) {
+  if (action.type === 'CHECK_STATUS') {
+    return 'Under Construction';
+  }
+  return state;
```

Comment:
- [ ] I have realized that you are using the ```CHECK_STATUS``` action type here but it is not being defined anywhere. Kindly add the check status action type and also use the duck's pattern on it as mentioned earlier.